lib/mzgl/app/apple/AppleWebView.mm

```diff
@@ -136,7 +136,7 @@ - (void)webView:(WKWebView *)webView
 
 - (void)userContentController:(nonnull WKUserContentController *)userContentController
 	  didReceiveScriptMessage:(nonnull WKScriptMessage *)message {
-	if ([message.name isEqual:@"closeWindow"]) {
+	if ([message.name isEqual:@"closeWindow"] || [message.body isEqual:@"close"]) {
```

Comment:
This was so aggravating to catch. Basically this can be sent by the view itself, for example if you destroy it 🙄